daml-lf/spec/transaction.rst

```diff
@@ -181,7 +181,7 @@ later.
 +--------------------+-----------------+
 |                 12 |      2021-01-27 |
 +--------------------+-----------------+
-|       (preview) 13 |      2021-04-06 |
+|                 13 |      2021-04-06 |
```

Comment:
do we track the dates when we make it a preview or when we make it stable?

Reply:
I do not know. :man_shrugging:
Dates in the specs are a mess.

Reply:
I try to not change the date once introduced.